var/spack/repos/builtin/packages/perl/package.py

```diff
@@ -93,6 +93,23 @@ def configure_args(self):
             '-Dloclibpth=' + self.spec['gdbm'].prefix.lib,
         ]
 
+        # Extensions are installed into their private tree via
+        # `INSTALL_BASE`/`--install_base` (see [1]) which results in a
+        # "predictable" installation tree that sadly does not match the
+        # Perl core's @INC structure.  This means that when activation
```

Comment:
What does Perl's core `@INC` structure look like? Perhaps we should be installing `PerlPackage`'s differently? Another possibility is to override `activate` and `deactivate` to copy the files to a differently directory. Just alternatives to this technique, not saying one would be better than the other.